neo/rawio/tdtrawio.py

```diff
@@ -197,7 +197,8 @@ def _parse_header(self):
                 for seg_index, segment_name in enumerate(segment_names):
                     # get data index
                     tsq = self._tsq[seg_index]
-                    mask = (tsq['evtype'] == EVTYPE_STREAM) & \
+                    mask = ((tsq['evtype'] == EVTYPE_STREAM) | \
+                           (tsq['evtype'] == int(33041))) & \
```

Comment:
Instead of of hardcoded I would create a global vriable for this number ssomething like 
`EVTYPE_STREAM_VARIANT=33041` to avoid hidden code.